examples/ucp_hello_world.c

```diff
@@ -575,8 +583,13 @@ int main(int argc, char **argv)
     status = ucp_worker_create(ucp_context, &worker_params, &ucp_worker);
     CHKERR_JUMP(status != UCS_OK, "ucp_worker_create\n", err_cleanup);
 
-    status = ucp_worker_get_address(ucp_worker, &local_addr, &local_addr_len);
-    CHKERR_JUMP(status != UCS_OK, "ucp_worker_get_address\n", err_worker);
+    worker_attr.field_mask = UCP_WORKER_ATTR_FIELD_ADDRESS;
+
+    status = ucp_worker_query(ucp_worker, &worker_attr);
+    CHKERR_JUMP(status != UCS_OK, "ucp_worker_query\n", err_worker);
+
+    local_addr     = worker_attr.address;
+    local_addr_len = worker_attr.address_length;
```

Comment:
maybe the variables local_addr, local_addr_len could be removed now?

Reply:
Yes, we can do this, but both this variable are also used inside `run_ucx_client` function. So I see three option how to deal with it:
1. Leave it as it done now
2. Make `worker_attr` a global variable. I don't think that this is a good option.
3. Pass `worker_attr` to `run_test` and therefore to `run_ucx_client` to obtain address information there. Maybe even remove the `run_test` wrapper at all.

I would prefer to implement the third option. Do you agree with me?

Reply:
option3 + remove run_test + pass local_addr+local_addr_len as **parameters** to run_ucx_client (instead of having them as global vars or passing ucp_worker_attr_t* parameter)